src/quo2/components/notifications/activity_logs_photos/view.cljs

```diff
@@ -1,14 +1,24 @@
 (ns quo2.components.notifications.activity-logs-photos.view
   (:require [react-native.core :as rn]
+            [quo2.components.markdown.text :as quo2.text]
```

Comment:
We can use it as' text' since we don't have to impose quo2 again in the alias.

Reply:
Okay